CONTRIBUTING.md

```diff
@@ -191,72 +193,75 @@ priority/p3 - Bugs that SHOULD be fixed by a given release, time permitting.
 
 priority/p4 - Bugs that SHOULD be fixed in a future (to be determined) release.
 
-### Backlog
+### Not Ready
 
-Issues in Backlog are not a current focus. For example, they may be feature requests or ideas for a future version of
-your project.
+The Not Ready column is for issues that need more discussion, details and/or triaging before being put in the [Backlog](#backlog). Issues in Not Ready should have assignee(s) to track whose input is needed to put the issue in the Backlog. For issues reported by VIC engineers: if the issue's details aren't fleshed out, the reporter should set themselves as the assignee.
+
+### Backlog
 
-When moving issues to the Backlog, add more information (like requirements and outlines) into each issue. It’s useful to
+Issues in Backlog should be ready to be worked on in future sprints. For example, they may be feature requests or ideas for a future version of
+the project. When moving issues to the Backlog, add more information (like requirements and outlines) into each issue. It's useful to
 get ideas out of your head, even if you will not be touching them for a while.
 
+To move an issue into the Backlog swim lane, it must have:
+
+1. a `priority/...` label
+2. a `team/...` label
+3. an estimated level of effort (see the table below for guidance for mapping effort to story points)
+4. no assignee (assignees are set when the issue is selected to work on)
+
+Story Points | Story Size
+------------ | -------------------------------------------------------
+1            | Less than 1 day of effort
+2            | 2 - 3 days of effort
+3            | 3 - 5 days of effort
+5            | 5 - 10 days of effort, consider splitting this if it's 7 - 10 days
+8            | > 10 days, anything of this size should be split before moving into In Progress
+
```

Comment:
This is fundamentally not a good idea for several reasons, we should stick with our existing definitions of story points or we are not using story points at all and need to move to time based estimates ( which aren't good, thus Agile uses story points )

Reply:
I see. My change doesn't add this table, it's already there: https://github.com/vmware/vic/blob/master/CONTRIBUTING.md#in-progress. I've just moved it from the In Progress section to the Backlog section.

Reply:
Should we remove this table altogether?

Reply:
I would like to see it corrected to our agreed upon estimation process:
```
Story points
● We have not been consistent in story point estimation
● Proposal:
– Stick with fibonacci
– All bugs are a 2 unless we know it is significantly more or less work than the
average bug
– 1 is easier than the average bug
– 3 is slightly more work than the average bug and probably should be about
an average feature work for an easy feature (which includes design doc,
implementation, testing, review)
– 5 is about 2x more work than the average bug and the highest single issue
value we want
– >5 can be on EPICs, but single issues should be decomposed further if you
think they are more than a 5
```

Reply:
Fixed: https://github.com/anchal-agrawal/vic/blob/860c9b267b8c59d9dc6057613bfc982213a787e7/CONTRIBUTING.md#story-point-estimates